examples/python/human_pose_tracking/README.md

```diff
@@ -16,9 +17,227 @@ channel = "main"
   <img src="https://static.rerun.io/human_pose_tracking/37d47fe7e3476513f9f58c38da515e2cd4a093f9/full.png" alt="">
 </picture>
 
-Use the [MediaPipe](https://google.github.io/mediapipe/) Pose solution to detect and track a human pose in video.
+# Overview
+
+Use the [MediaPipe](https://google.github.io/mediapipe/)  Pose Landmark Detection solutions to track human body pose in images and videos.
+
+Logging Details:
+
+1. Human Body Pose Landmarks as 2D Points:
+
+   - Specify connections between the points
+
+   - Extracts hand landmark points as 2D image coordinates.
+
+   - Logs the 2D points to the Rerun SDK.
+   
+
+2. Human Body Pose Landmarks as 3D Points:
+
+    - Specify connections between the points
+
+    - Detects body pose landmarks using MediaPipe solutions.
+
+    - Converts the detected body pose landmarks into 3D coordinates.
+
+    - Logs the 3D points to the Rerun SDK.
```

Comment:
Any reason for the empty lines for each item?

Reply:
Some inconsistencies:
- Period / no period at end
- Inconsistent tense for verbs